features-json/css-grid-animation.json

```
@@ -0,0 +1,490 @@
+			"9.1":"n",
+			"10":"n",
+			"10.1":"n",
+			"11":"n",
+			"11.1":"n",
+			"12":"n",
+			"12.1":"n",
+			"13":"n",
+			"13.1":"n",
+			"14":"n",
+			"14.1":"n",
+			"15":"n",
+			"15.1":"n",
+			"15.2-15.3":"n",
+			"15.4":"n",
+			"15.5":"n",
+			"16.0":"y",
+			"TP":"y"
+		},
+		"opera":{
+			"9":"n",
+			"9.5-9.6":"n",
+			"10.0-10.1":"n",
+			"10.5":"n",
+			"10.6":"n",
+			"11":"n",
+			"11.1":"n",
+			"11.5":"n",
+			"11.6":"n",
+			"12":"n",
+			"12.1":"n",
+			"15":"n",
+			"16":"n",
+			"17":"n",
+			"18":"n",
+			"19":"n",
+			"20":"n",
+			"21":"n",
+			"22":"n",
+			"23":"n",
+			"24":"n",
+			"25":"n",
+			"26":"n",
+			"27":"n",
+			"28":"n",
+			"29":"n",
+			"30":"n",
+			"31":"n",
+			"32":"n",
+			"33":"n",
+			"34":"n",
+			"35":"n",
+			"36":"n",
+			"37":"n",
+			"38":"n",
+			"39":"n",
+			"40":"n",
+			"41":"n",
+			"42":"n",
+			"43":"n",
+			"44":"n",
+			"45":"n",
+			"46":"n",
+			"47":"n",
+			"48":"n",
+			"49":"n",
+			"50":"n",
+			"51":"n",
+			"52":"n",
+			"53":"n",
+			"54":"n",
+			"55":"n",
+			"56":"n",
+			"57":"n",
+			"58":"n",
+			"60":"n",
+			"62":"n",
+			"63":"n",
+			"64":"n",
+			"65":"n",
+			"66":"n",
+			"67":"n",
+			"68":"n",
+			"69":"n",
+			"70":"n",
+			"71":"n",
+			"72":"n",
+			"73":"n",
+			"74":"n",
+			"75":"n",
+			"76":"n",
+			"77":"n",
+			"78":"n",
+			"79":"n",
+			"80":"n",
+			"81":"n",
+			"82":"n",
+			"83":"n",
+			"84":"n",
+			"85":"n",
+			"86":"n",
+			"87":"n"
+		},
+		"ios_saf":{
+			"3.2":"n",
+			"4.0-4.1":"n",
+			"4.2-4.3":"n",
+			"5.0-5.1":"n",
+			"6.0-6.1":"n",
+			"7.0-7.1":"n",
+			"8":"n",
+			"8.1-8.4":"n",
+			"9.0-9.2":"n",
+			"9.3":"n",
+			"10.0-10.2":"n",
+			"10.3":"n",
+			"11.0-11.2":"n",
+			"11.3-11.4":"n",
+			"12.0-12.1":"n",
+			"12.2-12.5":"n",
+			"13.0-13.1":"n",
+			"13.2":"n",
+			"13.3":"n",
+			"13.4-13.7":"n",
+			"14.0-14.4":"n",
+			"14.5-14.8":"n",
+			"15.0-15.1":"n",
+			"15.2-15.3":"n",
+			"15.4":"n",
+			"15.5":"n",
+			"16.0":"n"
+		},
+		"op_mini":{
+			"all":"n"
+		},
+		"android":{
+			"2.1":"n",
+			"2.2":"n",
+			"2.3":"n",
+			"3":"n",
+			"4":"n",
+			"4.1":"n",
+			"4.2-4.3":"n",
+			"4.4":"n",
+			"4.4.3-4.4.4":"n",
+			"101":"n"
+		},
+		"bb":{
+			"7":"n",
+			"10":"n"
+		},
+		"op_mob":{
+			"10":"n",
+			"11":"n",
+			"11.1":"n",
+			"11.5":"n",
+			"12":"n",
+			"12.1":"n",
+			"64":"n"
+		},
+		"and_chr":{
+			"102":"n"
+		},
+		"and_ff":{
+			"101":"y"
+		},
+		"ie_mob":{
+			"10":"n",
+			"11":"n"
+		},
+		"and_uc":{
+			"12.12":"n"
+		},
+		"samsung":{
+			"4":"n",
+			"5.0-5.4":"n",
+			"6.2-6.4":"n",
+			"7.2-7.4":"n",
+			"8.2":"n",
+			"9.2":"n",
+			"10.1":"n",
+			"11.1-11.2":"n",
+			"12.0":"n",
+			"13.0":"n",
+			"14.0":"n",
+			"15.0":"n",
+			"16.0":"n"
+		},
+		"and_qq":{
+			"10.4":"n"
+		},
+		"baidu":{
+			"7.12":"n"
+		},
+		"kaios":{
+			"2.5":"n"
+		}
+	},
+	"notes":"",
+	"notes_by_num":{
+
+	},
+	"usage_perc_y":0,
+	"usage_perc_a":0,
+	"ucprefix":false,
+	"parent":"parentfeatureid",
```

```suggestion
	"parent":"",
```

Searching for the error I ended up at
https://github.com/Fyrd/caniuse/blob/c9309d571180baf8eac87833fa7c6b6754d3129a/validator/validate-jsons.js#L238-L242

While it's strange that the other PRs (at least #6317) didn't fail with that, it seems like the only place it could come from!?
Soooo maybe lets try my suggestion, emptying that? :D

---

Removed on the same branch.